src/main/webapp/help-snapshotPolicyId.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    (Optional) The ID of the snapshot policy for the VDB.
```

Comment:
The policy type is called `VDB Snapshot` 
so...
>(Optional) The ID of the VDB Snapshot policy for the VDB.

https://cd.delphix.com/docs/latest/policies-for-scheduled-jobs